cmd/amppkg_dl_sxg/main.go

```diff
@@ -90,11 +99,25 @@ func main() {
 	if err != nil {
 		log.Fatalf("%+v", err)
 	}
+	cURL, err := url.Parse(certURL)
+	if err != nil {
+		log.Fatalf("%+v", err)
+	}
+	fURL, err := url.Parse(*flagCertURL)
```

Comment:
Maybe move this url.Parse call inside the `if *flagCertURL`?

Reply:
Fixed.